lib/datasource/pod/index.ts

```diff
@@ -98,14 +101,17 @@ const githubRegex = regEx(
   /^https:\/\/github\.com\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
 );
 
+const githubEnterpriseRegex = regEx(
+  /(?<hostURL>(^https:\/\/github\..*\.com))\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
```

Comment:
Thanks, @ranolf for the suggestion:

```suggestion
  /(?<hostURL>(^https:\/\/github\..+\.[a-zA-z0-9-]+))\/(?<account>[^/]+)\/(?<repo>[^/]+?)(\.git|\/.*)?$/
```